lib/src/widgets/icon_button_with_text.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
```

Comment:
Make sure to add SPDX-FileCopyrightText and SPDX-License-Identifier to every new file.

Reply:
done